packages/dev-middleware/package.json

```diff
@@ -18,25 +18,12 @@
   "files": [
     "dist"
   ],
-  "scripts": {
-    "build": "yarn clean && babel src --out-dir dist",
-    "dev": "babel src --out-dir dist --source-maps --watch",
-    "clean": "rimraf dist",
-    "prepare": "yarn build"
-  },
```

Comment:
are we going to publish this package? If yes, how do we want to provide these scripts and devDeps. to consumer. Context: There's no `"private": true` field...

Reply:
Yes this package will be published (and depended on in upcoming changes). These `scripts` are now managed at the repo level.

Reply:
Good to know. 👍